sdk/config_helpers.go

```diff
@@ -897,3 +917,8 @@ func GetAppProtectPolicyAndSecurityLogFiles(cfg *proto.NginxConfig, ignoreDirect
 
 	return policies, profiles
 }
+
+// note: ito ignore directives use GetAppProtectPolicyAndSecurityLogFiles
```

Comment:
You can probably remove the `note:` comments as they are a bit unnecessary

Reply:
I was told to add them, but can remove I don't mind @oliveromahony

Reply:
I think the comment is valid, you don't need the note part :)